README.md

```diff
@@ -45,18 +49,35 @@ git clone https://github.com/Arcadia-Science/B-SOID.git
 cd /path/to/B-SOID/
 ```
 
-For MacOS users
-```
+For MacOS users (note: you need an x86 archicture MacOS):
```

Comment:
or M1 pro with rosetta? i think that is what my laptop is and things seemed to work fine?